sycl/ReleaseNotes.md

```diff
@@ -1,3 +1,1595 @@
+# Release notes Jul'24
+
+Release notes for commit range
+[d2817d6d317db1](https://github.com/intel/llvm/commit/d2817d6d317db1143bb227168e85c409d5ab7c82)
+...
+[ebb3b4a21b3b0e](https://github.com/intel/llvm/commit/ebb3b4a21b3b0e977f44434781729df7de83e436)
+
+## TODO
+
+commit https://github.com/intel/llvm/commit/9876e19f4ff387b35b0c98c7d62e5f50e6de187d
+    [SYCL][XPTI] 'queue_id' metadata feature refactoring (#13070)
+    bugfix?
+
+commit https://github.com/intel/llvm/commit/29b4d855fa1a378e89182795e0d368304c40c3f6
+    [SYCL][CUDA] Enable support of msvc math functions for nvptx target. (#14007)
+
+commit https://github.com/intel/llvm/commit/7a9d3b1e9483b69baa0b8c6f1097016efd52854c
+    [SYCL][NVPTX] Do not decompose SYCL functor unless necessary (#14434)
+
+commit https://github.com/intel/llvm/commit/38e663ecd37de513d8e31afdfdf245cf8c9d17f0
```

Comment:
@againull, I found that this patch helped to resolve some weird issue, but do you have any more meaningful insights into it in context of mentioning it in release notes?

Reply:
Decided to completely drop this one in 82e564153684b5597ce075032cbbc8b2218a2d3f. Let me know if there are objections to that